README.md

```diff
@@ -36,7 +36,12 @@ terraform apply ./terraform.tfplan
 ## Testing
 
 ```
-go test ./...
+# Unit
+make test
+
+# Integration
+cp <my-google-credentials-file> ./credentials.json
```

Comment:
can we mention where/how to get this credentials file?

Reply:
Updated